clarity-ext-scripts/clarity_ext_scripts/covid/sminet_client.py

```diff
@@ -474,14 +478,70 @@ def create_covid_request(created, sample_info, referring_clinic, patient):
     :patient: An object created with the Patient factory
     """
 
+    if not created:
+        created = datetime.datetime.now()
+
     reporting_doctor = Doctor("Lars Engstrand")
```

Comment:
Should this be hard-coded? Feels like this, and the `lab_number`, might be better to send as info, to the client at least. Even if it is probably not of great relevance here I'd just feel more comfortable if the hard coding was at least done on script rather than client level.

Reply:
For simplicity I just hardcoded it as it's the only requirement for now (who hasn't said that before), but I agree that it should not be in the client!